### PR TITLE
Chore: Add JS4Girls

### PR DESCRIPTION
**Pull request by harsh183**
Based on https://github.com/cristianoliveira/awesome4girls/issues/137

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,11 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 ### Javascript
 ([Go to top](#awesome4girls---))
 
+  - [Js4Girls](http://webschool.io/js4girls/) - Loc: Brazil - - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+  
+  A Brazilian initiative to provide free javascript classes to women in a male-free environment.
+
+
```
Comment (cristianoliveira) [inline README.md]: There is an extra empty line here. I think it is causing the build fail

Comment (harsh183) [reply README.md]: Yes we have to, create a few more issues regarding that.

**Comment by cristianoliveira** [comment]
I was wrong :(

File: README.md [hunk 2]
```diff
@@ -61,6 +61,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 ### Javascript
 ([Go to top](#awesome4girls---))
 
+  - [Js4Girls](http://webschool.io/js4girls/) - Loc: Brazil - - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+  
+  A Brazilian initiative to provide free javascript classes to women in a male-free environment.
```
Comment (cristianoliveira) [inline README.md]: I found the problem. Here must have 4 spaces. I will add it on the fail description

**Pull request merged**